nncf/common/quantization/quantizer_propagation/graph.py

```diff
@@ -1498,6 +1502,14 @@ def _get_weight_and_activation_qconfig_list_intersection(
                     act_qconfig_updated.signedness_to_force = signedness_to_force_position
                     act_qconfig_extend_list.append(act_qconfig_updated)
         act_qconfig_extend_list += activation_qconfig_options
+
+        activation_qconfig_narrow_range_true = []
+        for act_qconfig in act_qconfig_extend_list:
+            if not act_qconfig.narrow_range:
+                act_qconfig_updated = deepcopy(act_qconfig)
+                act_qconfig_updated.narrow_range = True
+                activation_qconfig_narrow_range_true.append(act_qconfig_updated)
+        act_qconfig_extend_list += activation_qconfig_narrow_range_true
```

Comment:
Activation configs are merged with the embedding weights configs regardless the `narrow_range` attribute